hack/e2e/run-rp-and-e2e.sh

```diff
@@ -65,9 +65,31 @@ register_sub() {
 }
 
 run_e2e() {
-    RESOURCEGROUP=$ARO_RESOURCEGROUP make test-e2e
+    export E2E_ENTRYPOINT="docker run --privileged \
+                --network host \
+                --env-file <(env | egrep 'AZURE|E2E|ARO') \
```

Comment:
env-file and env: ick.  Let's just write this out the long way.

Reply:
The long way is VERY long, and if you add new var and forget to update here - flakes incoming :/ if all our vars would have the same prefix, of few - we could use only one `--env-file`